completions/jarsigner

```diff
@@ -25,33 +25,34 @@ _jarsigner()
             return
             ;;
         -signedjar)
-            _filedir jar
+            _filedir '@(jar|apk)'
             return
             ;;
     esac
 
     # Check if a jar was already given.
     local i jar=false
     for (( i=0; i < ${#words[@]}-1; i++ )) ; do
-        if [[ "${words[i]}" == *.jar && \
-            "${words[i-1]}" != -signedjar ]]; then
+        if [[ "${words[i]}" == *.jar || "${words[i]}" == *.apk && \
+            "${words[i-1]}" != -signedjar ]] ; then
             jar=true
             break
         fi
     done
 
-    if ! $jar ; then
-        if [[ "$cur" == -* ]]; then
+    if ! ($jar || $apk) ; then
```

Comment:
Doesn't seem anytihing sets `$apk` nor is necessary, so this change should just be removed.